modules/network-observability-multitenancy.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-multi-tenancy{context}"]
+= Enable multi-tenancy in Network Observability
+Multi-tenancy in the Network Observability Operator allows and restricts individual user access, or group access, to the flows stored in Loki. Access is enabled for project admins. Project admins who have limited access to some namespaces can access flows for only those namespaces.
+
+.Procedure
+. Authorize reading permission to `user1` by running the following command: 
++
+[source, terminal]
+----
+$ oc adm policy add-cluster-role-to-user netobserv-reader user1
+----
++
+Now, the allowed users have access to the Network Flows Traffic page in the console, which is the same privilege as cluster admins, with data restricted to their allowed namespaces. For example, a user that has access to a single namespace can see all the flows internal to this namespace, as well as flows going from and to this namespace.
```

Comment:
```suggestion
Now, the data is restricted to user allowed namespaces. For example, a user that has access to a single namespace can see all the flows internal to this namespace, as well as flows going from and to this namespace.
```

The first part of the sentence is confusing to me as we only apply roles on the storage here. The access to the page is not related to this procedure.

Reply:
Addressed and reworked this sentence.